cogs/conversioncog.py

```diff
@@ -0,0 +1,27 @@
+from discord.ext import commands
+from unit_converter.converter import convert, converts
+
+
+class ConversionCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot  # This line in the constructor is required
+
+    @commands.command()
+    async def convert(
+        self, ctx, inputunit, outputunit
+    ):  # the ctx parameter is short for 'context'
+        """type your input and output units"""

```

Comment:
Is this an accurate description of how the command works? It seems as though the input isn't just a unit, but is actually a quantity with a unit. Remember that although you have read the documentation or examples for the `convert` function you imported from `unit-converter`, your users have not, and neither have other programmers who may be viewing and working with your code. Make sure you use variable names and include documentation that is accurate to the way your code works! :smile: